### PR TITLE
Fix sig-cli meeting recordings url

### PR DESCRIPTION
**Pull request by budhrg**
Previously some random video was playing on clicking the hyperlink for meeting recordings.

### CODE REVIEW

**Comment by nikhita** [comment]
cc @fabianofranz

**Comment by cblecker** [comment]
This needs to be fixed in sigs.yaml, and then a "make" run: https://github.com/kubernetes/community/blob/f1b01d1b432bc120a3609bad0afe456a4b612ab2/sigs.yaml#L221

**Comment by budhrg** [comment]
> This needs to be fixed in sigs.yaml, and then a "make" run:

Sure. Thanks

File: sig-list.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ When the need arises, a [new SIG can be created](sig-creation-procedure.md)
...
-|[Product Management](sig-product-management/README.md)|* [Aparna Sinha](https://github.com/apsinha), Google<br>* [Ihor Dvoretskyi](https://github.com/idvoretskyi), CNCF<br>* [Caleb Miles](https://github.com/calebamiles), CoreOS<br>|* [Slack](https://kubernetes.slack.com/messages/kubernetes-pm)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-pm)|* [Tuesdays at 15:00 UTC (biweekly)](https://zoom.us/j/845373595)<br>
+|[OpenStack](sig-openstack/README.md)|* [Ihor Dvoretskyi](https://github.com/idvoretskyi), Mirantis<br>* [Steve Gordon](https://github.com/xsgordon), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-openstack)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-openstack)|* [Wednesdays at 15:00 UTC (biweekly)](https://zoom.us/j/417251241)<br>
```
Comment (budhrg) [inline sig-list.md]: Found after doing `make`.

Comment (cblecker) [reply sig-list.md]: Sorry.. this was caused by another PR. You should be able to rebase, and it will go away

Comment (budhrg) [reply sig-list.md]: Gone as you said @cblecker . Thanks.

**Pull request merged**